### ISSUE TITLE
fail to update sage from 9.4 to 9.6 due to file conflicts

### ISSUE THREAD

**Issue by liangjs** [cschwan/sage-on-gentoo#698]
```
 * Messages for package sci-mathematics/sage_setup-9.6:

 * This package will overwrite one or more files that may belong to other
 * packages (see list below). You can use a command such as `portageq
 * owners / <filename>` to identify the installed package that owns a
 * file. If portageq reports that only one package owns a file then do
 * NOT file a bug report. A bug report is only useful if it identifies at
 * least two or more packages that are known to install the same file(s).
 * If a collision occurs and you can not explain where the file came from
 * then you should simply ignore the collision since there is not enough
 * information to determine if a real problem exists. Please do NOT file
 * a bug report at https://bugs.gentoo.org/ unless you report exactly
 * which two packages install the same file(s). See
 * https://wiki.gentoo.org/wiki/Knowledge_Base:Blockers for tips on how
 * to solve the problem. And once again, please do NOT file a bug report
 * unless you have completely understood the above message.
 * 
 * Detected file collision(s):
 * 
 * 	/usr/lib/python3.9/site-packages/sage_setup/excepthook.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/run_parallel.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/util.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/cython_options.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/extensions.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/library_order.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/setenv.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/clean.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/find.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_ext_minimal.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_cython.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_ext.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_install.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext_minimal.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext_minimal.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/__init__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_cython.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_cython.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_install.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_cython.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_install.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext_minimal.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_install.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/giacpy-mkkeywords.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__main__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/storage.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/utils.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/memory.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/generator.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/instructions.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__main__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/rdf.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/cdf.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/cc.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/element.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/python.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/rr.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/base.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cc.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cdf.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/base.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rdf.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cdf.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/python.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/base.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/python.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/element.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cc.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/__init__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/element.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/element.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rdf.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rr.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cdf.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rr.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rdf.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/cc.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/python.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/rr.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/specs/__pycache__/base.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/generator.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/instructions.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/utils.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/utils.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/instructions.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__init__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/generator.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__main__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/storage.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/generator.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/utils.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/storage.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__main__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/memory.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/__main__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/storage.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/memory.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/instructions.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/interpreters/__pycache__/memory.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__init__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__main__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__main__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/giacpy-mkkeywords.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/giacpy-mkkeywords.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/__main__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/autogen/__pycache__/giacpy-mkkeywords.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/setenv.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/util.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/excepthook.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/find.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/cython_options.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/util.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/extensions.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/util.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/library_order.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/run_parallel.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/find.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/__init__.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/excepthook.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/clean.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/run_parallel.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/find.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/setenv.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/setenv.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/cython_options.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/library_order.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/library_order.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/run_parallel.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/extensions.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/extensions.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/clean.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/excepthook.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/cython_options.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/__pycache__/clean.cpython-39.opt-2.pyc
 * 
 * Searching all installed packages for file collisions...
 * 
 * Press Ctrl-C to Stop
 * 
 * sci-mathematics/sage-9.4:0::sage-on-gentoo
 * 	/usr/lib/python3.9/site-packages/sage_setup/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/clean.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__init__.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/__init__.cpython-39.opt-2.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/__init__.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build.cpython-39.opt-1.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/__pycache__/sage_build_ext_minimal.cpython-39.pyc
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_cython.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_ext.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_build_ext_minimal.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/command/sage_install.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/cython_options.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/excepthook.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/extensions.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/find.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/library_order.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/run_parallel.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/setenv.py
 * 	/usr/lib/python3.9/site-packages/sage_setup/util.py
 * 
 * Package 'sci-mathematics/sage_setup-9.6' NOT merged due to file
 * collisions. If necessary, refer to your elog messages for the whole
 * content of the above message.
```

**Comment by kiwifb** [comment]
I didn't expect that kind of upgrade. I have been toying with `pkgchk` and I follow its recommendations to remove stuff and I should have waited. A work around is to unmerge the old sage first. But I will include a fix in `sage_setup` and in `sage_docbuild` which should show the same issue.

**Comment by kiwifb** [comment]
fixed by 10e05b7747defcd7a791bd2a684674ffef8cad90 and dbf949de55280263555ac0c22cb4b6abb682d933 - sync the overlay and try again. Do not hesitate to fill more issues, I can only fix problems I know about.
